src/test/java/uk/gov/hmcts/reform/professionalapi/repository/PrdEnumRepositoryTest.java

```diff
@@ -40,4 +40,12 @@ void test_findAll_manageOrg_roles() {
         assertThat(prdEnums.get(45).getEnumName()).isEqualTo("caseworker-employment");
         assertThat(prdEnums.get(46).getEnumName()).isEqualTo("caseworker-employment-legalrep-solicitor");
     }
+
+    @Test
+    void test_findAll_manageOrg_employment_roles() {
```

Comment:
Should this name not refer to privatelaw rather than employment.

Reply:
Matt, I have amended the test method name, please could you check and resolve?